source/guide/index.md

```diff
@@ -10,124 +10,125 @@ version: 2.1
+Weex provides a command line tool, the [weex-toolkit](/tools/toolkit.html), to help developers get start easily. The CLI can help you create a starter project, setup iOS and Android development environments, debug, install plugins, and so on.
 
-Currently, the `weex-toolkit` only supports the creation of Vue.js project. The `rax-cli` maybe helpful if you want to use Rax. Please visit [Rax's official website](https://alibaba.github.io/rax/) for more details.
+Currently, the `weex-toolkit` only supports the creation of Vue.js project. The `rax-cli` may be helpful if you want to use Rax. Please visit [Rax's official website](https://alibaba.github.io/rax/) for more details.
 
 ### Set up
 
-Make sure you have already installed the [Node.js](https://nodejs.org/), and then install the `weex-toolkit` globally.
+With [Node.js](https://nodejs.org/) installed, install `weex-toolkit` CLI globally.
 
 ```bash
 npm install weex-toolkit -g
 ```
 
-It will add the `weex` command to your global path. Then you can use `weex create` to create a starter project:
+This will add the `weex` command to your global path, and will allow you to generate new projects with the `weex create <project-name>` command.
+Use `weex create` to create a starter project:
 
 ```bash
 weex create awesome-app
 ```
 
-After doing that, you'll get a standard **Weex + Vue.js** project under the `awesome-app` folder of current path.
+After doing that, a standard **Weex + Vue.js** project will be generated inside the `awesome-app` folder in the current path.
 
 ### Develop
 
-The next step is to navigate into the generated directory, install dependencies, and start.
+The next step is to navigate into the generated directory, install dependencies, and start:
 
 ```bash
 cd awesome-app
 npm install
 npm start
 ```
 
-It will start a web server on the `8081` port. You can open `http://localhost:8081` to see the render result on the web. The source code is located in `src/` folder. You can develop it as normal Vue.js project.
+`npm start` will start a web server on port `8081`. Open `http://localhost:8081` in your browser of choice to see the rendered result of your Weex app. The source code is located in `src/` folder. You can develop it as normal Vue.js project.
 
 ![Preview](./images/toolkit-preview.png)
 
-Besides, You can also open `http://localhost:8081/web/preview.html` to preview the rendered result for web in iframe. You can also scan the QR code generated on the right using the [Weex playground app](/tools/playground.html) then you'll see the rendered result on the mobile device.
+Additionally, you can open `http://localhost:8081/web/preview.html` to preview the rendered result on the web in an iframe. You can also scan the QR code generated on the right using the [Weex playground app](/tools/playground.html) then you'll see the rendered result on the mobile device.
```

Comment:
"You can also scan the QR code generated on the right using the [Weex playground app](/tools/playground.html) then you'll see the rendered result on the mobile device." can be changed to "You can also scan the QR code generated on the right using the [Weex playground app](/tools/playground.html) to see the rendered result on the mobile device."